pkg/subctl/cmd/show_versions.go

```diff
@@ -58,17 +58,9 @@ func getOperatorVersion(clientSet kubernetes.Interface, versions []versionImageI
 	}
 
 	operatorFullImageStr := operatorConfig.Spec.Template.Spec.Containers[0].Image
-	img := strings.Split(operatorFullImageStr, ":")
-	var image string
-	var version string
-	if len(img) < 2 {
-		image = img[0]
-	} else {
-		image = img[0]
-		version = img[1]
-	}
+	i := strings.LastIndex(operatorFullImageStr, ":")
```

Comment:
If ':' isn't found, `i` will be -1 which causes a panic. This **shouldn't** happen but wouldn't hurt to handle as it did before.